fe/fe-core/src/main/java/org/apache/doris/catalog/RefreshManager.java

```diff
@@ -19,26 +19,44 @@
 
 import org.apache.doris.analysis.CreateTableStmt;
 import org.apache.doris.analysis.DropTableStmt;
+import org.apache.doris.analysis.RefreshCatalogStmt;
 import org.apache.doris.analysis.RefreshDbStmt;
 import org.apache.doris.analysis.RefreshTableStmt;
 import org.apache.doris.analysis.TableName;
 import org.apache.doris.catalog.external.ExternalDatabase;
 import org.apache.doris.common.DdlException;
+import org.apache.doris.common.ThreadPoolManager;
 import org.apache.doris.common.UserException;
 import org.apache.doris.datasource.CatalogIf;
 import org.apache.doris.datasource.ExternalCatalog;
 import org.apache.doris.datasource.ExternalObjectLog;
 import org.apache.doris.datasource.InternalCatalog;
+import org.apache.doris.qe.DdlExecutor;
 
+import com.google.common.collect.Maps;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 
 import java.util.Map;
+import java.util.concurrent.ScheduledThreadPoolExecutor;
+import java.util.concurrent.TimeUnit;
 
 // Manager for refresh database and table action
-public class RefreshManager {
+public class RefreshManager implements Runnable {
     private static final Logger LOG = LogManager.getLogger(RefreshManager.class);
 
+    private static final ScheduledThreadPoolExecutor REFRESH_TIMER = ThreadPoolManager.newDaemonScheduledThreadPool(1,
+        "catalog-refresh-timer-pool", true);
+    // Unit:SECONDS
+    private static final int REFRESH_TIME = 5;
+    // key is the name of a catalog, value is an array of length 2, used to store
+    // the original refresh time and the current remaining time of the catalog
+    private Map<String, Integer[]> refreshMap = Maps.newConcurrentMap();
+
+    public RefreshManager() {
+        REFRESH_TIMER.scheduleAtFixedRate(this::run, 0, REFRESH_TIME, TimeUnit.SECONDS);
```

Comment:
Should not start the schedule thread in constructor.
Because if do so, every time a `RefreshManager` is created, the schedule thread will start.
But the schedule thread should only start on Master FE.
So you need to add a new method `start()` to start schedule thread.
But this `start()` method should be called in `Env.startMasterOnlyDaemonThreads()`

Reply:
Ok, I thought `RefreshManager` was a singleton object before such an bug occurred, I will correct it. And add `start()` in `Env.startMasterOnlyDaemonThreads()`